### PR TITLE
COPS-2995 Replaced Spartan references with dcos-net

### PR DESCRIPTION
**Pull request by SarahStegall**
## Description

https://jira.mesosphere.com/browse/COPS-2995

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by SarahStegall** [comment]
Replaced all references to Spartan with dcos-net. 
Standardized metadata excerpts and titles.
Versions changed:
1.11 ent and oss
1.10 ent and oss
1.9 ent and oss
1.8 ent and oss

**Comment by GoelDeepak** [comment]
@SarahStegall we just have to change Spartan reference in 1.11 ent and oss. Not in previous releases.

File: pages/1.9/installing/ent/troubleshooting/index.md [hunk 1]
```diff
@@ -107,25 +107,25 @@ enterprise: false
...
             ```
-        
+
             **Note:** Running this command in multi-master configurations can take up to 10-15 minutes to complete. If it doesn't complete after 10-15 minutes, you should carefully review the `journalctl -flu dcos-exhibitor` logs.
 
 1.  Verify whether you can ping the DNS Forwarder (`ready.spartan`). If not, review the DNS Dispatcher service logs: ﻿⁠⁠⁠⁠
 
     ```bash
-    journalctl -flu dcos-spartan﻿⁠⁠⁠⁠
+    journalctl -flu spartan
```
Comment (GoelDeepak) [inline pages/1.9/installing/ent/troubleshooting/index.md]: shouldn't this be dcos-spartan?

File: pages/1.8/administration/installing/ent/custom/troubleshooting/index.md [hunk 2]
```diff
@@ -14,76 +14,76 @@ enterprise: false
...
     adjtimex -p
     timedatectl
     ```
 
 1. Ensure that firewalls and any other connection-filtering mechanisms are not interfering with cluster component communications. TCP, UDP, and ICMP must be permitted.
-    
-   Ensure that services that bind to port `53`, which is required by DNS Dispatcher (`dcos-spartan.service`), are disabled and stopped. For example: 
-   
+
+   Ensure that services that bind to port `53`, which is required by DNS Dispatcher (`docs-spartan.service`), are disabled and stopped. For example:
```
Comment (GoelDeepak) [inline pages/1.8/administration/installing/ent/custom/troubleshooting/index.md]: minor typo ... it should be `dcos` not `docs`

File: pages/1.11/installing/oss/troubleshooting/index.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ oss: true
 
 1. Ensure that firewalls and any other connection-filtering mechanisms are not interfering with cluster component communications. TCP, UDP, and ICMP must be permitted.
 
-   Ensure that services that bind to port `53`, which is required by DNS Forwarder (`dcos-spartan.service`), are disabled and stopped. For example:
+   Ensure that services that bind to port `53`, which is required by DNS Forwarder (`dcos-dcos-net.service`), are disabled and stopped. For example:
```
Comment (GoelDeepak) [inline pages/1.11/installing/oss/troubleshooting/index.md]: minor typo ... it should be dcos-net and not dcos-dcos-net

File: pages/1.11/installing/ent/troubleshooting/index.md [hunk 4]
```diff
@@ -107,25 +107,25 @@ enterprise: false
...
     -  Review the Mesos-DNS service logs with this command: ﻿
 
        ```bash
        ⁠⁠⁠⁠journalctl -flu dcos-mesos-dns﻿⁠⁠⁠⁠
        ```
-    
-    -  If you are able to ping `ready.spartan`, but not `leader.mesos`, review the Mesos master service logs by using this command: 
-    
+
+    -  If you are able to ping `ready.dcos-net`, but not `leader.mesos`, review the Mesos master service logs by using this command:
```
Comment (GoelDeepak) [inline pages/1.11/installing/ent/troubleshooting/index.md]: it should be `spartan`

File: pages/1.11/installing/ent/troubleshooting/index.md [hunk 5]
```diff
@@ -107,25 +107,25 @@ enterprise: false
                 }
             ]
             ```
-        
+
             **Note:** Running this command in multi-master configurations can take up to 10-15 minutes to complete. If it doesn't complete after 10-15 minutes, you should carefully review the `journalctl -flu dcos-exhibitor` logs.
 
-1.  Verify whether you can ping the DNS Forwarder (`ready.spartan`). If not, review the DNS Dispatcher service logs: ﻿⁠⁠⁠⁠
+1.  Verify whether you can ping the DNS Forwarder (`ready.dcos-net`). If not, review the DNS Dispatcher service logs: ﻿⁠⁠⁠⁠
```
Comment (GoelDeepak) [inline pages/1.11/installing/ent/troubleshooting/index.md]: it should be spartan. `ready.spartan` is a DNS name. We haven't change the DNS names

File: pages/1.9/installing/ent/troubleshooting/index.md (hunk 1 above)
Comment (SarahStegall) [reply pages/1.9/installing/ent/troubleshooting/index.md]: Fixed

File: pages/1.8/administration/installing/ent/custom/troubleshooting/index.md (hunk 2 above)
Comment (SarahStegall) [reply pages/1.8/administration/installing/ent/custom/troubleshooting/index.md]: Fixed!

File: pages/1.11/installing/oss/troubleshooting/index.md (hunk 3 above)
Comment (SarahStegall) [reply pages/1.11/installing/oss/troubleshooting/index.md]: Fixed!

File: pages/1.11/installing/ent/troubleshooting/index.md (hunk 4 above)
Comment (SarahStegall) [reply pages/1.11/installing/ent/troubleshooting/index.md]: Fixed!

File: pages/1.11/installing/ent/troubleshooting/index.md (hunk 5 above)
Comment (SarahStegall) [reply pages/1.11/installing/ent/troubleshooting/index.md]: Fixed!

File: pages/1.11/installing/oss/troubleshooting/index.md [hunk 6]
```diff
@@ -106,10 +106,10 @@ oss: true
 
             **Note:** Running this command in multi-master configurations can take up to 10-15 minutes to complete. If it doesn't complete after 10-15 minutes, you should carefully review the `journalctl -flu dcos-exhibitor` logs.
 
-1.  Verify whether you can ping the DNS Forwarder (`ready.spartan`). If not, review the DNS Dispatcher service logs: ﻿⁠⁠⁠⁠
+1.  Verify whether you can ping the DNS Forwarder (`ready.dcos-net`). If not, review the DNS Dispatcher service logs: ﻿⁠⁠⁠⁠
```
Comment (GoelDeepak) [inline pages/1.11/installing/oss/troubleshooting/index.md]: This should remain `spartan`

File: pages/1.11/installing/oss/troubleshooting/index.md [hunk 7]
```diff
@@ -120,7 +120,7 @@ oss: true
        ⁠⁠⁠⁠journalctl -flu dcos-mesos-dns﻿⁠⁠⁠⁠
        ```
 
-    -  If you are able to ping `ready.spartan`, but not `leader.mesos`, review the Mesos master service logs by using this command:
+    -  If you are able to ping `ready.dcos-net`, but not `leader.mesos`, review the Mesos master service logs by using this command:
```
Comment (GoelDeepak) [inline pages/1.11/installing/oss/troubleshooting/index.md]: This should remain `spartan`

**Comment by SarahStegall** [comment]
Approved by Seth.

**Pull request merged**